README.md

```diff
@@ -1,24 +1,60 @@
 # astronomer-airflow-version-check
 
-An Apache Airflow plugin that will periodically (default, once per day) check
-if there is a new version of Astronomer's Certified Airflow and display a
-message in the UI
+An Apache Airflow plugin that will periodically (default, once per day) check if there is a new version of Astronomer's Certified Airflow and display a message in the UI.
+
+Additionally, this plugin provides warnings in the Airflow UI for the following scenarios:
+- When the current version will reach its end of life (EOL) in 30 days (default, configurable).
+- When the current version has already reached its end of life (EOL).
+
+This plugin also provides options to dismiss the EOL warnings for a configurable number of days (default is 7 days). There is also an option to completely disable the EOL warning.
 
 ## Settings
 
-This plugin looks at the following settings under the `astronomer` section of
-the Airflow config. The easiest way of setting this is via environment
-variables prefixed with `AIRFLOW__ASTRONOMER__`
+This plugin looks at the following settings under the `astronomer` section of the Airflow config. The easiest way of setting this is via environment variables prefixed with `AIRFLOW__ASTRONOMER__`
```

Comment:
This line shouldnt show any difference, same for other lines which don't have any content diff other than `.`

Reply:
I think this is because of the indentation change in Readme. I have changed the indentation back to original.